docs/data-discovery/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Data Discovery"
```

Comment:
This header is unlike the others in two ways:
* It’s not using the gerund (“*Managing* data quality,” “*Modeling* your data”).
* It has the second word capitalized.

At the least, we should change to `Data discovery`. But if you are ok with `Discovering data`, `Exploring your data`, `Exploring your tracking` or something like that, it would bring even more consistency.

Reply:
done